client/verta/tests/models/standard_models.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+
+import pickle
+
+import pytest
+
+from verta.registry import VertaModelBase
+
+
+class VertaModel(VertaModelBase):
+    ARTIFACT_KEY = "artifact"
+
+    def __init__(self, artifacts):
+        with open(artifacts[self.ARTIFACT_KEY], "rb") as f:
+            self.artifact = pickle.load(f)
+
+    def predict(self, input):
+        return self.artifact
+
+
+def verta_models():
+    models = []
+
+    models.append(VertaModel)
+
+    return models
+
+
+def keras_models():
+    keras = pytest.importorskip("tensorflow.keras")
+    np = pytest.importorskip("numpy")
+
+    models = []
+
+    # TODO: re-enable with VR-11962
+    # # sequential API
+    # model = keras.Sequential(
+    #     [
+    #         keras.layers.Dense(3, activation="linear", name="layer1"),
+    #         keras.layers.Dense(2, activation="relu", name="layer2"),
+    #         keras.layers.Dense(1, activation="sigmoid", name="layer3"),
+    #     ]
+    # )
+    # model(np.random.random(size=(3, 3)))  # initialize weights
+    # models.append(model)
```

Comment:
I've commented out some models because they actually do cause test failures.

But because this PR is just simple wrappers over existing functionality, I don't personally consider them a reason to block this ticket.

I've split off investigating+resolving the values to VR-11962.